index.html

```diff
@@ -0,0 +1,156 @@
+    <script src="https://kit.fontawesome.com/555b0c0f91.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href=".\styles.css">
+    <!-- Link to bootstrap.min.css-->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css"><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/lato-font/3.0.0/css/lato-font.min.css">
+    <!-- Link to jquery and popper.min.js -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+    <!-- Link to bootstrap.min.js -->
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
+    <title>Interior Design.</title>
+</head>
+<body>
+    <section class="main-section">
+    <header class="navbar">
+        <nav class="navbar-1">
+            <i class="fa-brands fa-facebook-f"></i>
+            <i class="fa-brands fa-twitter"></i>
+            <a class="nav-items-11" href="#">English</a>
+            <a class="nav-items-12" href="#">My Page</a>
+            <a class="nav-items-13" href="#">Logout</a>
+        </nav>
+        <nav class="navbar-2">
+            <h2 class="nav-logo">SUAVE INTERIORS</h2>
+            <a class="nav-items" href="./index.html">Home</a>
+            <a class="nav-items" href="./about.html">About</a>
+            <a class="nav-items" href="#">Program</a>
+            <a class="nav-items" href="#">News</a>
+            <a class="nav-items-2" href="#">dc Campaign</a>
+        </nav>
+        <nav class="nav-list">
+            <a  class="nav-items" href="./index.html">Home</a>
+            <a  class="nav-items" href="./about.html">About</a>
+            <a class="nav-items" href="#">Program</a>
+            <a class="nav-items" href="#">News</a>
+            <a class="nav-items-2" href="#">dc Campaign</a>
+        </nav>
+            <div class="bars">
+              <div class="bars-one"></div>
+              <div class="bars-one"></div>
+              <div class="bars-one"></div>
+            </nav>
+          </nav>
+        </header>
+        <section id="home-page">
+        <div id="home-page" class="title1">
+            <p class="firstline">"Hello! Interior Design Lovers"</p>
+            <h3 class="header">We bring to you the 2022<br> 
+                Interior design Symposium live in Uganda. </h3>
+        </div>
+        <div class="quote">
+            <p class="quote-1">We bring to you great speakers from all around the world to <br>
+                boast you knowlegdge and understanding of interior designs.<i class="fas fa-broadcast-tower    "></i>
+                 </p>
+        </div>
+        <div class="details">
+            <p class="details-1">2022.10.12(FRI) ~ 12(SAT)</p>
+               <p class="details-2"> @ lugogo Cricket oval.</p>
+        </div>
+    </section>
```

Comment:
- [ ] Kindly make sure that You follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including: Colors, Layout: composition and space between elements.
- [ ] For class="title1" `h3` Kindly adjust color just like the design 
![Capt1](https://user-images.githubusercontent.com/93598189/191448666-ed3d6974-df3c-4ca8-85c9-d6e80e79a0de.PNG)

Your project Design Look likes this : 
![Captur12](https://user-images.githubusercontent.com/93598189/191448789-fbbbc8c8-404a-426d-9172-df4bc7fc88fd.PNG)

- [ ] Kindly also adjust Layout: composition and space between elements for class="details" of second `p` it just overlapped by the second section just show above on image.

Reply:
- [ ] This still needs work to be done. The text should be aligned as requested by the previous CR.